### PR TITLE
Add support for LMS_INTERNAL_ROOT_URL.

### PR DESCRIPTION
**Pull request by mtyaka**
This adds ability to configure the `LMS_INTERNAL_ROOT_URL` setting via the `EDXAPP_LMS_INTERNAL_ROOT_URL` ansible variable. The value of the new variable is used instead of `EDXAPP_LMS_ROOT_URL` to construct enterprise API URLs, which are used for internal communication. In some deployment scenarios, it is not desirable for internal communication to be carried out via the external URLs.

This change is backwards compatible since the new `EDXAPP_LMS_INTERNAL_ROOT_URL` defaults to the value of `EDXAPP_LMS_ROOT_URL`.

Support for `LMS_INTERNAL_ROOT_URL` is being added in https://github.com/edx/edx-platform/pull/16510 and https://github.com/edx/edx-enterprise/pull/240.

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @mtyaka! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?number=4200&repo=edx%2Fconfiguration"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

**Review by UmanShahzad — COMMENTED** [review]
Looks good! Will do formal testing when we're able.

File: playbooks/roles/automated/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# edX Configuration
+#
+# github:     https://github.com/edx/configuration
+# wiki:       https://openedx.atlassian.net/wiki/display/OpenOPS
+# code style: https://openedx.atlassian.net/wiki/display/OpenOPS/Ansible+Code+Conventions
+# license:    https://github.com/edx/configuration/blob/master/LICENSE.TXT
+#
+
+# Allow this role to be duplicated in dependencies.
+allow_duplicates: yes
```
Comment (feanil) [inline playbooks/roles/automated/meta/main.yml]: @mtyaka looks like this commit was pulled in from your other PR.

Comment (mtyaka) [reply playbooks/roles/automated/meta/main.yml]: Thanks @feanil, I pulled that commit into this branch for testing - I removed it now.

**Review by UmanShahzad — APPROVED** [review]
:+1:

- [x] I tested this: Saw that applying these variables correctly makes them show up in the environment file.
- [x] I read through the code
- [x] I checked for accessibility issues
- [x] Includes documentation

**Comment by mtyaka** [comment]
Thanks @UmanShahzad!

@feanil Could your review this?

**Review by feanil — APPROVED** [review]

**Comment by mtyaka** [comment]
Rebased from 03d57437eb5f42ddb09b6b291a6ca4fe599c3575.

**Pull request merged**